rasa/core/broker.py

```diff
@@ -223,3 +225,52 @@ def _publish(self, event):
 
     def _close(self):
         self.producer.close()
+
+
+class SQLProducer(EventChannel):
+    from sqlalchemy.ext.declarative import declarative_base
+
+    Base = declarative_base()
+
+    class SQLEvent(Base):
```

Comment:
Should we name that differently to avoid confusion with the `SQLTrackerStore.SQLEvent`?

Reply:
Yes! You're right. I'll change it to `SQLBrokerEvent`.